warp10/src/main/java/io/warp10/standalone/StandaloneAcceleratedStoreClient.java

```diff
@@ -0,0 +1,304 @@
+package io.warp10.standalone;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.RejectedExecutionException;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.concurrent.locks.LockSupport;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.warp10.WarpConfig;
+import io.warp10.continuum.Configuration;
+import io.warp10.continuum.TimeSource;
+import io.warp10.continuum.gts.GTSDecoder;
+import io.warp10.continuum.gts.GTSEncoder;
+import io.warp10.continuum.store.Constants;
+import io.warp10.continuum.store.DirectoryClient;
+import io.warp10.continuum.store.GTSDecoderIterator;
+import io.warp10.continuum.store.MetadataIterator;
+import io.warp10.continuum.store.StoreClient;
+import io.warp10.continuum.store.thrift.data.DirectoryRequest;
+import io.warp10.continuum.store.thrift.data.Metadata;
+import io.warp10.quasar.token.thrift.data.ReadToken;
+import io.warp10.quasar.token.thrift.data.WriteToken;
+
+public class StandaloneAcceleratedStoreClient implements StoreClient {
+  
+  private static final Logger LOG = LoggerFactory.getLogger(StandaloneAcceleratedStoreClient.class);
+  
+  private final StoreClient persistent;
+  private final StandaloneChunkedMemoryStore cache;
```

Comment:
So it must be a chunked memory store. Warp 10 must fail or warn if not properly configured.
Maybe the most clean will be to have `accelerator.chunk.count` and `accelerator.chunk.length` configurations.